modules/op-installing-pipelines-as-code-cli.adoc

```diff
@@ -9,13 +9,13 @@
 [role="_abstract"]
 Cluster administrators can use the `tkn pac` and `opc` CLI tools on local machines or as containers for testing. The `tkn pac` and `opc` CLI tools are installed automatically when you install the `tkn` CLI for {pipelines-title}.
 
-You can install the `tkn pac` and `opc` version `1.10.0` binaries for the supported platforms:
+You can install the `tkn pac` and `opc` version `1.11.0` binaries for the supported platforms:
 
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.10.0/tkn-linux-amd64.tar.gz[Linux (x86_64, amd64)]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.10.0/tkn-linux-s390x.tar.gz[Linux on {ibmzProductName} and {linuxoneProductName} (s390x)]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.10.0/tkn-linux-ppc64le.tar.gz[Linux on {ibmpowerProductName} (ppc64le)]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.10.0/tkn-macos-amd64.tar.gz[macOS]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.10.0/tkn-windows-amd64.zip[Windows]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.11.0/tkn-linux-amd64.tar.gz[Linux (x86_64, amd64)]
```

Comment:
1.10 binaries are uploaded on URL with both `pipeline` and `pipelines` in path. We could update it to `pipelines` in this doc for 1.11, it's closer to official product name.